modules/aioseop_sitemap.php

```diff
@@ -1875,7 +1875,7 @@ public function get_sitemap_index_filenames() {
 				$options[ "{$this->prefix}taxonomies" ] = array();
 			}
 			$options[ "{$this->prefix}posttypes" ]  = array_diff( $options[ "{$this->prefix}posttypes" ], array( 'all' ) );
-			$options[ "{$this->prefix}taxonomies" ] = array_diff( $options[ "{$this->prefix}taxonomies" ], array( 'all' ) );
+			$options[ "{$this->prefix}taxonomies" ] = apply_filters( "{$this->prefix}show_taxonomy", array_diff( $options[ "{$this->prefix}taxonomies" ], array( 'all' ) ) );
```

Comment:
Filter is missing documentation; there are other implementations of this filter missing docs.

Instead of having multiple apply_filters of the same filter, would it be better to reduce it to calling a function to apply the filter?